### DISCUSSION TITLE
Semantic Search & OpenAI

### DISCUSSION THREAD

**Discussion by alecmarcus** [zed-industries/zed#6609] [Feature Requests]
I'd love to try semantic search, but I don't feel comfortable accepting this without being able to clearly tell the security team in what capacity our code is shared with OpenAI. Specifically, whether the code will be ingested into any models or stored in its original form somewhere.

![CleanShot 2023-10-25 at 13 55 13@2x](https://github.com/zed-industries/community/assets/18580335/c9662f36-c9b4-4525-b2bf-85662129d539)

I'm guessing it's to create project-specific [embeddings](https://platform.openai.com/docs/guides/embeddings). Is that true? Would love some insight here.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
